cheatsheets/Server_Side_Request_Forgery_Prevention_Cheat_Sheet.md

```diff
@@ -378,6 +378,12 @@ In cloud environments SSRF is often used to access and steal credentials and acc
 
 To leverage this protection migrate to IMDSv2 and disable old IMDSv1. Check out [AWS documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instancedata-data-retrieval.html) for more details.
 
+## Semgrep Rules
+
+[Semgrep](https://semgrep.dev/) is a command-line tool for offline static analysis. Use pre-built or custom rules to enforce code and security standards in your codebase.
+Checkout the Semgrep rule for SSRF to identify/investigate for SSRF vulnerabilities in Java
+[https://semgrep.dev/salecharohit:owasp_java_ssrf](https://semgrep.dev/salecharohit:owasp_java_ssrf)
```

Comment:
I see that this link is to semgrep.dev/salecharohit:owsap_java_ssrf will it change after they merge your PR? Should we wait with this PR until they merge it master?